.github/workflows/ci.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        ruby-version: [2.2.10, 2.3.8, 2.4.10, 2.5.9, 2.6.9, 2.7.5, 3.0.3, 3.1.0, jruby-9.3.3.0]
+        ruby-version: [2.2.10, 2.3.8, 2.4.10, 2.5.9, 2.6.9, 2.7.5, 3.0.3, 3.1.0, jruby-9.3.4.0]
```

Comment:
`variable-mapper` needs to have its jruby version updated, too.

Reply:
Great catch, @olleolleolle! Much appreciated. Addressed with [aac5f67](https://github.com/newrelic/newrelic-ruby-agent/pull/1035/commits/aac5f67604eceb0e589431b9f2f05abc085c69f2)